website/docs/docs/build/sl-getting-started.md

```diff
@@ -26,14 +26,21 @@ This getting started page recommends a workflow to help you get started creating
 New to dbt or metrics? Try our [Jaffle shop example project](https://github.com/dbt-labs/jaffle-sl-template) to help you get started!
 :::
 
+## Install MetricFlow
+
+Before you begin, make sure you install the `metricflow` and [dbt adapter](/docs/supported-data-platforms) via PyPI in the CLI using the `pip install "dbt-metricflow[your_adapter_name]"` command. This is because you're installing MetricFlow as an extension of the dbt adapter. Currently, the supported adapters are Snowflake and Postgres (BigQuery, Databricks, and Redshift coming soon).
```

Comment:
Nit: The second sentence here feels disjointed from the first. I think this is because it isn't apparent what "This is because" is in reference to. Perhaps something like 

> Note that specifying `[you_adapter_name]` is required. This is because you must install MetricFlow as an extension of a dbt adapter.

Reply:
Or maybe something like

> This irregular pip install is necessary because you're installing...

Reply:
ah fab, great feedback @QMalcolm ! how does this look:

>Before you begin, make sure you install the `metricflow` and [dbt adapter](/docs/supported-data-platforms) via PyPI in the CLI. To install them, open the command line interface (CLI) and use the pip install command `pip install "dbt-metricflow[your_adapter_name]"`. Note that specifying `[your_adapter_name]` is required.  This is because you must install MetricFlow as an extension of a dbt adapter.

Reply:
@mirnawong1 I think that update would be perfect! 🙂